File: Me-supraja (1)/Me-supraja/CrEST/Microsoft.Crest.Web.API/CrEST.Data/Models/Application.cs
﻿using System;
using System.Collections.Generic;

namespace CrEST.Data.Models
{
    public partial class Application
    {
        public int ApplicationId { get; set; }
        public int? SoWid { get; set; }
        public string ServiceLine { get; set; }
        public string Application1 { get; set; }
        public int? ServiceClass { get; set; }
        public int? RunOrGrow { get; set; }
        public string ApplicationGroup { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public bool? EndToEnd { get; set; }
        public bool? Epm { get; set; }
        public string TM { get; set; }
        public string ManagedCapacity { get; set; }
        public string ManagedService { get; set; }
        public string SoftwareAssetSearchableId { get; set; }
        public string Remarks { get; set; }
        public int? SupplierId { get; set; }
        public string OwnerAlias { get; set; }
        public int? Itorg { get; set; }

        public virtual Itorg ItorgNavigation { get; set; }
        public virtual ServiceClass ServiceClassNavigation { get; set; }
        public virtual SoW SoW { get; set; }
    }
}

File: CrEST/Microsoft.CrEST.UI/Microsoft.CrEST.UI/angularCrest/app/sla/shared/models/application.ts
export class Application {
    //application: string
    //contactId: string
    //serviceLine: string
    //serviceClass: string
    //runVsGrow: string
    //applicatioGroup: string 
    //startDate: string
    //endDate: string
    //serviceLineChk: string
    //endToEnd: string
    //epm: string
    //tm: string
    //managedCapacity: string
    //managedServices: string
    //applicationChk:string
    //softwareAssetSearchableID: string
    //remarks: string

    applicationId: number;
    soWid: number;
    serviceLine: string;
    application1: string;
    serviceClass: number;
    runOrGrow: number;
    applicationGroup: string;
    startDate: Date;
    endDate: Date;
    endToEnd: boolean;
    epm: boolean;
    tM: string;
    managedCapacity: string;
    managedService: string;
    softwareAssetSearchableId: string;
    remarks: string;
    supplierId: number;
    ownerAlias: string;
    itorg: number;
    iTOrgName: number;
}

      
         
         //string Application1 { get; set; }
         //int ? ServiceClass { get; set; }
         //int ? RunOrGrow { get; set; }
         //string ApplicationGroup { get; set; }
         //DateTime ? StartDate { get; set; }
         //DateTime ? EndDate { get; set; }
         //bool ? EndToEnd { get; set; }
         //bool ? Epm { get; set; }
         //string TM { get; set; }
         //string ManagedCapacity { get; set; }
         //string ManagedService { get; set; }
         //string SoftwareAssetSearchableId { get; set; }
         //string Remarks { get; set; }
         //int ? SupplierId { get; set; }
         //string OwnerAlias { get; set; }
         //int ? Itorg { get; set; }

         //virtual Itorg ItorgNavigation { get; set; }
         //virtual ServiceClass ServiceClassNavigation { get; set; }
         //virtual SoW SoW { get; set; }